### DISCUSSION TITLE
Issue displaying a custom slot

### DISCUSSION THREAD

**Discussion by msteuper** [microsoft-search/pnp-modern-search#4097] [Q&A] [answered]
Hi all, 

I have an issue displaying a custom slot in a PnP Search result part. I have configured the the new slots in the section "Layout slots"
![image](https://github.com/user-attachments/assets/a11d3b17-03b6-4e7d-916d-1515ac061126)

I have ensured that the managed properties are configured in "Selected properties"
![image](https://github.com/user-attachments/assets/a39e8737-1a1a-4486-8147-aa17c500f93c)

I double checked in the debug result view that at least one property returns a value
![image](https://github.com/user-attachments/assets/ad2f198a-4901-400f-ae39-993d56473df6)

But when I customize the list view with the following snippet

                        <span>                            
                            <span class="template--listItem--author">
                                Process Group: {{slot item @root.slots.ProcessGroup}}
                            </span>                        
                        </span>   

I only get the following back - the value for the slot item is blank.
![image](https://github.com/user-attachments/assets/101397f9-ae5a-477f-8948-aa5603d3a47a)

Any hints or tips what I'm missing or what is going wrong are highly welcome.

Thanks a lot in advance
Mar

**Comment by kasperbolarsen** [comment]
It sure looks like it should work just fine.
Could you try to use the Details List layout and adding the 00GovernanceArea as a column, just to verify that the basic works

**Comment by msteuper** [comment]
Hi Kasper, thanks for your response. I already tried this and unfortunately the same. No values are shown in the detail list.

**Reply by kasperbolarsen** [reply]
OK,
Can you share the settings for the 00GovernanceArea managed property? Where was it mapped, on the tenant level or on a site collection?
Does a query like 00GovernanceArea:Supply* yield anything?

**Comment by msteuper** [comment]
it was configured on site level in the Refinablestring110
![image](https://github.com/user-attachments/assets/86f70a01-895e-427c-8cfe-fa0695f86075)

The query "00GovernanceArea:Supply*" provides 84 search results. So seems to work.

**Reply by kasperbolarsen** [reply]
Could you try to check if RefinableString110 is selected in Selected Properties, and use RefinableString110 in the Slots instead of 00GovernanceArea ?  (I have seen weird issues with aliases before)

**Comment by msteuper** [comment]
Dear Kasper,

unfortunately the same results. I addedd "RefineableString110" to the Selected Properties, I addedd a new slot and added a column in the details list view. No values are displayed. Also not when I filter the search for a specific value which results in one single search result.

**Reply by kasperbolarsen** [reply]
Perhaps looking into the data using one of the debugging tools from here ( https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/ ) could provide some insights?

Otherwise, you could sign up for a ½ hours review of the issue ( PnP Modern Search office hours) at [aka.ms/pnp/modernsearchofficehours](https://forms.office.com/pages/responsepage.aspx?id=xt4SFDF540m7kCquHMmhqrjqma_5-shDoiBP0hNegIRUMVhNN0ZDQ0FRWkVWT0lDUDBCOVdTR0xLMi4u)

**Comment by wobba** [comment, marked as answer]
A workaround is using the property directly `item.00GovernanceArea` instead of slots.

**Reply by msteuper** [reply]
Hi Mikael, 
thanks for the hint. that works!!
